tests/test_command_parse.py

```diff
@@ -59,6 +59,16 @@ def parse_item(self, response):
     def parse(self, response):
         return [scrapy.Item(), dict(nomatch='default')]
 
+class MyRetryRequestsSpider(CrawlSpider):
+    name = 'retryRequests{0}'
+    start_urls = ['https://httpbin.org/'] 
+
+    def parse(self, response):
+        if response.meta.get('retried'):
+            self.logger.info("done")
+            return
+        response.meta['retried'] = True
+        yield response.request.replace(dont_filter=True)
```

Comment:
It would be nice if we could extend this test a little. It should include yielded items as well as a non-default callback  (i.e. a request that has `callback` set to some method).